### PR TITLE
Improve Readme

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
Improve the readme and make its usage better and simple. 

## What
- Add an index to the Readme
- Organize the sections
- Fix some text errors. 
- Fix layout
- Adding pre-requirements to dev setup
- Change to use adoc since it shows the current standard defined by docs.

## Why
- Make the understanding clear
- Allow a README standard 
- Allow addressing its user to specific points and topics
- Make all projects across the org follow the same layout/standard of README. ( E.g https://github.com/integr8ly/installation )

## Verification Steps
Check its result [here](https://github.com/integr8ly/tutorial-web-app/blob/8fbf470489d8ae169b45c64f36053cc90afa7279/README.adoc)

## Checklist:

- [X] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member

## Progress

- [x] Finished task
- [ ] TODO

### CODE REVIEW

**Comment by camilamacedo86** [comment]
@finp it is not a complex markdown. It is just using the default GitHub stile to MD.

**Comment by camilamacedo86** [comment]
Hi @finp.

Ok. I will do it for here and the installation project and I will ping you.

**Comment by davidkirwan** [comment]
Big :+1: on asciidoc for me anyway.

**Comment by camilamacedo86** [comment]
Hi @finp and @davidkirwan,

I am totally against to use adoc format for many reasons. However, I also think that it is a decision that should be discussed for ALL and not unilaterally. I will raise a thread internally for we are able to discuss this until Monday.

**Comment by camilamacedo86** [comment]
@davidkirwan and @finp 

I give another chance for asciidoc and yes folks you were correct it is really good and no so hard as I thought at the first moment. However, I am stiil not able to add an automatic index on top of the doc and I don't know why the tip/note images are not showing as in its docs.

**Comment by davidkirwan** [comment]
https://gist.github.com/dcode/0cfbf2699a1fe9b46ff04c41721dda74#table-of-contents wonder does this work @camilamacedo86 add this to the top

----
:toc:
toc::[]
----

**Comment by davidkirwan** [comment]
@camilamacedo86 yes this works see: https://raw.githubusercontent.com/davidkirwan/advent_of_code/master/README.asciidoc

for an example, no need to use doctoc at all, adding those commands at the top of the file will enable the TOC generation automatically.

**Comment by camilamacedo86** [comment]
Hi @davidkirwan did not work the index and the images from the adoc style, however, I think we can go forward and after try to fix it. Wdyt?

File: README.adoc [hunk 1]
```diff
@@ -1,45 +1,58 @@
-# tutorial-web-app
+= Tutorial Web App
+
+:toc:
```
Comment (davidkirwan) [inline README.adoc]: Put up at the very top @camilamacedo86

Comment (davidkirwan) [reply README.adoc]: Above the = Tutorial Web App

Comment (davidkirwan) [reply README.adoc]: Or if you want the toc to appear under the first heading then:

At the very top do:
:toc:
:toc-placement!:

and then where you want the TOC to appear do:
toc::[]

**Comment by camilamacedo86** [comment]
@davidkirwan and @finp 

Now with index and images. I can say that now I love adoc :-)

Really tks.

**Comment by davidkirwan** [comment]
@camilamacedo86 @finp another convertee to Asciidoc! :tada:

**Pull request merged**